### DISCUSSION TITLE
Issue "this user account is blocked."

### DISCUSSION THREAD

**Discussion by xuannam12** [vercel/community#4823] [Account Issue]
### Question

I registered an account with my Gitlab. But I received the message "This
user account is blocked" and can't access my account. Can you check it for
me?
I sent an email after but your team has not had contact with me for 2
moths. My account still blocked.
( my case number is 00162677 )

**Comment by KierynMo** [comment]
How are they not able to get to this? I have been blocked for over a month with no support
![Screenshot 2023-11-28 at 3 42 25 pm](https://github.com/vercel/community/assets/78433837/491ae749-62df-40a1-a2ee-f422ee2f74f6)
